chcemvediet/apps/inforequests/templates/inforequests/admin/action_delete_confirmation.html

```diff
@@ -0,0 +1,13 @@
+{% extends "admin/delete_confirmation.html" %}
+{% load amend prepend from poleno.amend %}
+
+{% block content %}
+  {% amend %}
+    {{ block.super }}
+    {% if object.type == object.TYPES.EXPIRATION or object.type == object.TYPES.APPEAL_EXPIRATION %}
+      {% prepend path=".//form" %}
+        <label><input name="snooze" type="checkbox" />Extend snooze of previous action.</label>
```

Comment:
V admine to nie je nijako nastylovane, aspon nejaky margin by sa hodil.
Tiez vo formularoch boli checkboxy este obalene v samostanom divku. 
```
<div class="checkbox">
<label><input type="checkbox" /> Some text.</label>
</div>
```
Dodrzat takuto sablonu a mam pridat este `<div>`?

Reply:
Hej, skus sa s tym nejak pohrat, nech to vyzera ako tak pekne. Nejake tie marginy a tak. Idelne pouzitim rovnakej struktiry ako ma admin defaultne. Cize skus ten div, alebo nieco take podobne.

Reply:
:ok:
Dal by sa spravit aj hack, ze by sa iba pridala medzera pred text.

Reply:
@viliambalaz To zmenu v css vyhod. Je to teraz nejak divne poposuvane vertikalne.

Ano, kludne daj medzeru medzi `<input>` a text. A pre vertikalny spacing staci namiesto `<div>` zabalit ten widget do `<p>`

Reply:
:ok: